src/components/ItemFilter.vue

```diff
@@ -40,26 +40,62 @@
         />
       </b-form-group>
 
+      <div class="additionalFilters">
+        <b-form-group label="Select additional field filters" label-for="availableFields" description="Fields advertised by the /queryables endpoint">
```

Comment:
I'd remove the description (or make it less technical). /queryables is a technical details normal users don't care about.